src/error_handling.c

```diff
@@ -33,6 +33,10 @@ extern "C"
 // RCUTILS_REPORT_ERROR_HANDLING_ERRORS and RCUTILS_WARN_ON_TRUNCATION are set in the header below
 #include "./error_handling_helpers.h"
 
+#ifndef SIZE_MAX
```

Comment:
include `<stdint.h>` for `SIZE_MAX`? On what platforms is it undefined?

Reply:
> include <stdint.h> for SIZE_MAX
yes, good catch

>  On what platforms is it undefined?
It's a C99 definition, I think the only place we don't get C99 is on Windows.

Reply:
It looks like VS 2017 and 2019 has `SIZE_MAX` in `<limits.h>`. https://docs.microsoft.com/en-us/cpp/c-runtime-library/data-type-constants?view=vs-2017 . Do you know if the "C Run-Time Library" is available when building C++ code on windows? If so, maybe the `ifndef` can be replaced with another include.

Reply:
I don't know offhand if it's available, but I can try a CI run on Windows without the `ifndef` to see if it's okay that way